x-pack/plugins/apm/public/components/app/diagnostics/summary_tab/index.tsx

```diff
@@ -6,27 +6,44 @@
  */
 
 import React from 'react';
-import { EuiFlexGroup, EuiCallOut } from '@elastic/eui';
+import {
+  EuiFlexGroup,
+  EuiCallOut,
+  EuiDescriptionList,
+  EuiSpacer,
+} from '@elastic/eui';
+
+import { APIReturnType } from '../../../../services/rest/create_call_apm_api';
 import { ApmIntegrationPackageStatus } from './apm_integration_package_status';
 import { IndexTemplatesStatus } from './index_templates_status';
 import { FieldMappingStatus } from './indicies_status';
 import { DataStreamsStatus } from './data_streams_status';
 import { useDiagnosticsContext } from '../context/use_diagnostics';
 
+type DiagnosticsBundle = APIReturnType<'GET /internal/apm/diagnostics'>;
+
 export function DiagnosticsSummary() {
   const { diagnosticsBundle } = useDiagnosticsContext();
 
-  const isCrossCluster = Object.values(
-    diagnosticsBundle?.apmIndices ?? {}
-  ).some((indicies) => indicies.includes(':'));
+  const hasLimitedPrivileges =
+    diagnosticsBundle &&
+    diagnosticsBundle.diagnosticsPrivileges.hasAllPrivileges === false;
+
+  const isCrossCluster = getIsCrossCluster(diagnosticsBundle);
 
-  if (isCrossCluster) {
+  if (isCrossCluster || hasLimitedPrivileges) {
     return (
-      <EuiCallOut title="Cross cluster search not supported" color="warning">
-        The APM index settings is targetting remote clusters. Please note: this
-        is not currently supported by the Diagnostics Tool and functionality
-        will therefore be limited.
-      </EuiCallOut>
+      <>
+        {isCrossCluster && (
```

Comment:
Do you really need to check the flags twice?

Reply:
Yes. In the first check it's either `isCrossCluster` or `!hasAllPrivileges `